pkg/utils/context.go

```diff
@@ -47,6 +47,7 @@ var (
 
 // SendingContextFrom creates the context to use when sending a Cloud Event with cloudevents.Client. It
 // sets the target if specified, and attaches a filtered set of headers from the initial request.
+// Deprecated: migrate to cloudevents sdk v2
```

Comment:
We don't need to migrate these. They're not needed anymore in v2 because the sdk doesn't have anymore HTTPTransportContext

Reply:
Same question about deprecation policy.